### PR TITLE
Adding clarification on SERVER CERTIFIATE and SERVER ASSYMETRIC KEY

### PR DESCRIPTION
**Pull request by PiJoCoder**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@PiJoCoder : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/t-sql/statements/backup-transact-sql.md [hunk 1]
```diff
@@ -325,6 +325,8 @@ If you choose to encrypt you will also have to specify the encryptor using the e
 - SERVER CERTIFICATE = Encryptor_Name
 - SERVER ASYMMETRIC KEY = Encryptor_Name
 
+The SERVER CERTIFICATE and SERVERY ASSYMETRIC KEY are a certificate and an assymetric key created in the Master database. For more informaiton see [CREATE CERTIFICATE](../../t-sql/statements/create-certificate-transact-sql.md) and [CREATE ASYMMETRIC KEY](../../t-sql/statements/create-asymmetric-key-transact-sql.md) respectively.
```
Comment (srutzky) [inline docs/t-sql/statements/backup-transact-sql.md]: Several grammatical issues should be fixed before merging this. The fixed version is:

The SERVER CERTIFICATE and SERVER ASYMMETRIC KEY are a certificate and an asymmetric key created in the `master` database. For more information see [CREATE CERTIFICATE](../../t-sql/statements/create-certificate-transact-sql.md) and [CREATE ASYMMETRIC KEY](../../t-sql/statements/create-asymmetric-key-transact-sql.md), respectively.

**Comment by ktoliver** [comment]
@MikeRayMSFT - Could you review the proposed updates? Thanks.

**Comment by MikeRayMSFT** [comment]
Recreating in privater repository in order to provide edits.

**Comment by MikeRayMSFT** [comment]
@PiJoCoder  - thanks for submitting this. Your changes will be included soon in an update. @srutzky thank you again for the edit.

**Pull request closed without merging**